TASK1/output.php

```diff
@@ -0,0 +1,96 @@
+<?php
+require 'creds.php';
+// Check if submit is set in post method then display hello fullname.
+if (isset($_POST['submit'])) {
+  // Render hello fullname
```

Comment:
Every line does not need to be explained.
We just need to explain the business logic.